packages/text-bitmap/src/BitmapFont.ts

```diff
@@ -85,6 +150,15 @@ export class BitmapFont
          */
         this.chars = {};
 
+        /**
+         * The map of base page textures (i.e., sheets of glyphs).
+         *
+         * @member {object}
+         * @readonly
+         * @private
+         */
+        this.pagesTextures = pagesTextures;
```

Comment:
Since page <-> texture have a one-to-one meaning, I think it should be `pageTextures`. Just a thought - https://softwareengineering.stackexchange.com/a/344761/307206

Reply:
Sure will rename.